tools/misc/get_image_metas.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+"""Get test image metas on a specific dataset.
+
+This script provides two ways to get test image metas including non-distribute
+and distribute way. Here is an example to run this script.
+
+Example:
+    Use non-distribute way::
+
+         python tools/misc/get_image_metas.py ${CONFIG} \
+         --out ${OUTPUT FILE NAME}
+
+    Use distribute way::
+        python -m torch.distributed.launch --nproc_per_node=${GPUS} \
+        tools/misc/get_image_metas.py ${CONFIG} --out ${OUTPUT FILE NAME} \
+        --launcher pytorch
+"""
+import argparse
+import os
+import time
+
+import mmcv
+from mmcv import Config
+from mmcv.runner import get_dist_info, init_dist
+
+from mmdet.apis.test import collect_results_cpu
+from mmdet.datasets import build_dataloader, build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Collect image metas')
```

Comment:
If we simply want meta-information (img width/height) of the data, we do not need to build the dataset and data loader to do that. We can simply parse the annotations and use multi-process to accelerate this process.